gnss/gps_time.py

```diff
@@ -133,6 +130,10 @@ def gpst_to_utc(gpst):
       the UTC representation of the corresponding gpst.
 
     """
+    if isinstance(gpst, dict):
```

Comment:
Aligning function with docstring which says that the function should be able to take a dict.